src/pages/Detail/Detail.js

```diff
@@ -1,7 +1,113 @@
-import React from 'react';
-
+import React, { useState, useEffect } from 'react';
+import './Detail.scss';
+import DetailImage from './DetailComponent/DetailImages';
+import DetailSmallImage from './DetailComponent/DetailSmallImage';
+import TextInfo from './DetailComponent/TextInfo';
+import ColorImage from './DetailComponent/ColorImage';
+import SizeTag from './DetailComponent/SizeTag';
+import Button from './DetailComponent/Button';
 const Detail = () => {
-  return <div>fdsfds</div>;
+  const [detailSmallImage, setDetailSmallImage] = useState();
+  const [textData, setTextData] = useState();
+  const [colorData, setColorData] = useState();
+  const [sizeData, setSizeData] = useState();
+
+  useEffect(() => {
+    fetch('/data/detailSmallList.json')
+      .then(res => res.json())
+      .then(res => setDetailSmallImage(res))
+      .catch(e => console.error(e));
+  }, []);
+  useEffect(() => {
+    fetch('/data/textInfoData.json')
+      .then(res => res.json())
+      .then(res => setTextData(res))
+      .catch(e => console.error(e));
+  }, []);
+  useEffect(() => {
+    fetch('/data/detailColorList.json')
+      .then(res => res.json())
+      .then(res => setColorData(res))
+      .catch(e => console.error(e));
+  }, []);
+  useEffect(() => {
+    fetch('/data/sizeTag.json')
+      .then(res => res.json())
+      .then(res => setSizeData(res))
+      .catch(e => console.error(e));
+  }, []);
+
+  return (
+    <div className="container">
+      <span>남성복</span>
+      <div id="imageContainer">
+        <DetailImage />
+        <div className="smallImageList">
+          {detailSmallImage &&
+            detailSmallImage.map(Small => {
```

Comment:
인자명 camelCase 지켜주세요!
```suggestion
            detailSmallImage.map(small => {
```